server/routes/communityOffenderManager/caseload.test.ts

```diff
@@ -0,0 +1,52 @@
+import { createComCase } from '../../data/__testutils/testObjects'
+import CaseloadRoutes from './caseload'
+import { mockedDate, mockRequest, mockResponse } from '../__testutils/requestTestUtils'
+import { createMockCommunityOffenderManagerCaseloadService } from '../../services/__testutils/mock'
+import { parseIsoDate } from '../../utils/utils'
+import { ProbationUser } from '../../interfaces/hmppsUser'
+
+const offenderSummaryList = [createComCase({})]
+
+const communityOffenderManagerCaseloadService = createMockCommunityOffenderManagerCaseloadService()
+const req = mockRequest({})
+const res = mockResponse({
+  locals: {
+    user: {
+      deliusStaffIdentifier: 1,
+    },
+  },
+})
+
+let caseloadRoutes: CaseloadRoutes
+
+beforeEach(() => {
+  caseloadRoutes = new CaseloadRoutes(communityOffenderManagerCaseloadService)
+  communityOffenderManagerCaseloadService.getCommunityOffenderManagerCaseload.mockResolvedValue(offenderSummaryList)
+  mockedDate(new Date(2022, 6, 1))
+})
+
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
+describe('GET', () => {
+  it('should render list of licences for approval', async () => {
+    await caseloadRoutes.GET(req, res)
+    expect(communityOffenderManagerCaseloadService.getCommunityOffenderManagerCaseload).toHaveBeenCalledWith(
+      req.middleware.clientToken,
+      res.locals.user as ProbationUser,
+    )
+    expect(res.render).toHaveBeenCalledWith('pages/communityOffenderManager/caseload', {
+      caseload: [
+        {
```

Comment:
will have to update create link based on the next loading page.

Reply:
**nit:** might want to call it the reviewLink or something